spec/controllers/api/v1/movies_show_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
```

Comment:
All tests for controller actions should be in the same file named after the controller you're testing. `spec/controllers/api/v1/movies_controller_spec.rb` will have all tests for all controller actions, keeping everything together

Reply:
put them all in same file